src/main/java/org/apache/pirk/utils/SystemConfiguration.java

```diff
@@ -160,4 +138,74 @@ public static void resetProperties()
     clearProperties();
     initialize();
   }
+
+  /**
+   * Loads the properties from local properties file in the specified directory
+   * <p>
+   * Only files ending in '.properties' will be loaded
+   */
+  public static void loadPropsFromDir(String dirName)
+  {
+    File dir = new File(dirName);
+    File[] directoryListing = dir.listFiles();
```

Comment:
You could use File#list(FilenameFilter), but we are not expecting a huge number of files in the dir list, this works fine.

```
        File[] directoryListing = new File(dirName).listFiles(new FilenameFilter() {
            @Override
            public boolean accept(File dir, String name) {
                return name.endsWith(".properties");
            }
        });
```